TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/ObstacleCoursePath.java

```diff
@@ -52,6 +60,9 @@ public void runOpMode() {
 
         if(isStopRequested()) return;
 
+        // first chunk
+        drive.followTrajectory(trajectoryFirstPart);
```

Comment:
maybe change the trajectory names of your path to match where you're going on the field